Chirp/Pages/ChatsPage/ChatsTableViewCell.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ChatsTableViewCell.swift
+//  Chirp
+//
+//  Created by Ady on 6/7/23.
+//
+
+import UIKit
+
+class ChatsTableViewCell: UITableViewCell {
+    
+    // Constants
+    private let imageViewWidth: CGFloat = 52
+    private let labelHeight: CGFloat = 18
+    private let leadingTrailingAnchor: CGFloat = 8
+    static let identifier = "ChatsTableViewCell"
+    
+    lazy var friendImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    lazy var fullNameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Full Name"
+        label.font = UIFont.boldSystemFont(ofSize: 16)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var lastMessageLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Last Message"
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var lastMessageTimeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Last Message Time"
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var unreadMessageCounterLabel: UILabel = {
+        let label = UILabel()
+        label.text = "10"
+        label.textColor = .white
+        label.backgroundColor = .red
+        label.clipsToBounds = true
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
You can allow the label reduce font size if the text is long and then center alight the text
```suggestion
        label.translatesAutoresizingMaskIntoConstraints = false
        label.adjustsFontSizeToFitWidth = true
        label.textAlignment = .center
```

Reply:
applied suggested changes